lib/web_driver_client/json_wire_protocol_client/response_parser.ex

```diff
@@ -268,6 +269,18 @@ defmodule WebDriverClient.JSONWireProtocolClient.ResponseParser do
     end
   end
 
+  @spec parse_server_status(Response.t()) ::
+          {:ok, ServerStatus.t()} | {:error, UnexpectedResponseError.t()}
+  def parse_server_status(%Response{value: value}) when is_map(value) do
+    ready = Map.get(value, "ready", true)
```

Comment:
How about
```suggestion
  def parse_server_status(%Response{value: %{"ready" => ready}}) when is_boolean(ready) do
```
? I think it's safer than defaulting to `true` in Map.get

Reply:
I take that back. I don't see "ready" at all in the spec: https://github.com/SeleniumHQ/selenium/wiki/JsonWireProtocol#status

A 200 HTTP status would be better to indicate readiness here I think

Reply:
Unfortunately, phantomJS's payload doesn't include a "ready" key.

```json
{
    "sessionId": null,
    "status": 0,
    "value": {
        "build": {
            "version": "1.2.0"
        },
        "os": {
            "name": "linux",
            "version": "unknown",
            "arch": "64bit"
        }
    }
}
```

I figured as long as I get a response back with a valid JWP status code, it's probably ready. What do you think?

Reply:
so something like 
```suggestion
  def parse_server_status(%Response{value: value, http_response: %HTTPResponse{status: 200}}) when is_map(value) do
    ready = true
```

Reply:
I like your suggestion and allowed me to remove the entire `parse_server_status` function via this commit: 5895a31. I've found instances where JWP servers are inconsistent in the status codes they return (some return 200 all of the time), so I thought I'd just ensure the response had a successful JWP status code.

What do you think?

Reply:
I saw the updates. I think it makes sense. I am wondering how the WebDrivers respond when the server is not ready, with some random 5xx status code? Maybe it would be possible to detect that and return `false` in that scenario instead of an error, since it's not really an unexpected response. But current approach seems fine as well

Reply:
I'm not sure if the server will do anything other than refusing the connection and don't know how to get the server in a "not ready" state. Maybe the more this gets used, we might run across when it makes sense to return `ready: false` for JWP.